resources/packages/fluid_styled_content/13.4/src/Resources/Private/PageView/Pages/Example.html.twig

```diff
@@ -0,0 +1,26 @@
+<f:layout name="Default" />
+<f:section name="Main">
+
+    <div style="background-color: yellow; padding: 0.5em 1em;">
+        <dl>
+            <dt>Template file</dt>
+            <dd>
+                <code>packages/{{ package.extensionKey }}/Resources/Private/PageView/Pages/Default.html</code>
+            </dd>
+            <dt>Backend Configuration</dt>
+            <dd>
+                <code>packages/{{ package.extensionKey }}/Configuration/Sets/SitePackage/PageTsConfig/BackendLayouts/default.tsconfig</code>
+            </dd>
+        </dl>
+    </div>
+
+    <div class="stage">
```

Comment:
i would like to opt in here for a simple backendlayout with 1 column and content output. this should be a really simple package people can adjust. for the planned tutorial it would be great if we would have more columns

Reply:
Then we would have no difference between the default and Example Backend Layout. Should we get rid of backend layouts all together?

Reply:
I would keep it as something the user can edit (add things) and start from, but yes really simple, as we also do not provide any styling here.

Reply:
I did posted an example. However now the Example and Default Template would be the same.  And the backend layout of default, example and not having one at all would also all be the same. 

We could as well remove the concept of backend layouts and just teach them in the extended site package example. What do you think?